composer-dependency-analyser.php

```diff
@@ -0,0 +1,10 @@
+<?php
```

Comment:
Sorry to be a pain - could you please check out `.gitattributes` and make sure this is `export-ignore`d _(Along with any other files that should be excluded)_